integration/simulation/simulation_in_mem_test.go

```diff
@@ -24,11 +42,75 @@ func TestInMemoryMonteCarloSimulation(t *testing.T) {
 		SimulationTime:            15 * time.Second,
 		L1EfficiencyThreshold:     0.2,
 		L2EfficiencyThreshold:     0.32,
-		L2ToL1EfficiencyThreshold: 0.34,
+		L2ToL1EfficiencyThreshold: 0.5,
```

Comment:
once we sort the ganache block time setting, let's put this back

Reply:
done !